### ISSUE TITLE
Set up mkdocs site in Engineering CoP

### ISSUE THREAD

**Issue by fyliu** [hackforla/peopledepot#165]
### Overview

We need the Engineering documentation to be using mkdocs so that we can easily copy sections from the website or our docs over to it when needed.

The idea is to move generally-useful engineering docs from the website docs or our docs over to engineering so that other teams can reference them as well.

### Action Items

- [ ] Propose to Engineering CoP that they start hosting documentation that's common to different hackforla teams
  - note, the CoP lead at the time agreed to it but there's different people currently and we should ask again.
- [ ] Ask what repo is available and best for setting up mkdocs. The site url would be something like `https://hackforla.github.io/engineering/` if the repo name is `https/github.com/hackforla/engineering/`
- [ ] Create an mkdocs site for the repo and make a PR. Instructions below
- [ ] Check that they have a working mkdocs site
- [ ] Check and release hackforla/knowledgebase-content#120 from ice box

### Resources

- hackforla's `docker-mkdocs` [image](https://hub.docker.com/r/hackforlaops/mkdocs), [repo](https://github.com/hackforla/docker-mkdocs), and [instructions](https://hackforla.github.io/docker-mkdocs/user-guide/#add-it-to-a-project) on adding it to a project
- website's [how to resolve merge conflicts](https://hackforla.github.io/website-wiki/How-to-Resolve-Merge-Conflicts/) guide
- website's [resources for setting up mkdocs](https://github.com/hackforla/website-wiki)
- PD's old sphinx doc site's [page on git](https://fyliu.github.io/peopledepot-sphinx/tools/git.html)

**Comment by fyliu** [comment]
Maybe this should exist in Engineering instead? This was from the discussion where any guides that has to do with generic tools like git should be moved to the Engineering docs. I thought that it would be easier if we set up an mkdocs instance there so we can just copy/paste the content, even if it has some mkdocs-specific markup.

**Comment by fyliu** [comment]
Need to update the description with work to be done, like create a docker-compose file and add a service to load and run the mkdocs docker image from the hackforlaops dockerhub account.

**Comment by ExperimentsInHonesty** [comment]
@fyliu I agree that this is probably better suited for the engineering CoP.  My only concern is that if we move it there, no one will take it up.  What we really from Engineering is a place to put tutorials that are common to all teams.  The existing wiki for engineering only has 11 pages.  The Data Science CoP has a lot of tutorials on their wiki and that seems to work for them, without requiring additional overhead of managing mkdocks.  I recommend that we just set up a tutorial section on the wiki for engineering and be done with it.  Then you will be able to contribute content there whenever something is useful to all teams.
- Engineering wiki: https://github.com/hackforla/engineering/wiki/Resources
- Data Science wiki: https://github.com/hackforla/data-science/wiki/Tutorials

**Comment by fyliu** [comment]
@ExperimentsInHonesty I understand. The wiki is a more practical approach for teams that don't already have exposure to mkdocs.
